.tool-versions

```diff
@@ -0,0 +1 @@
+nodejs 16.14.0
```

Comment:
@kespinola what is this tool-versions? Maybe it's easier to keep track of node version in package.json?